simphony/cuds/particles.py

```diff
@@ -448,6 +449,19 @@ def _update_element(self, cur_dict, element, clone):
         else:
             raise ValueError('id: {} does not exist'.format(uid))
 
+# ================================================================
+
+    # properties
+
+# ================================================================
+    def _get_data(self):
+        return copy.deepcopy(self._data)
```

Comment:
I think using deepcopy here is probably an overkill. Using `DataContainer(self._data)` should be enough. There is only one possible failure mode when the `values` in the dictionary are mutable objects like list or numpy arrays. However, I thought that we do not use them (if we do then it should probably be an error).